### ISSUE TITLE
Error in running xmeshfem3d command

### ISSUE THREAD

**Issue by VarunD21029** [SPECFEM/specfem3d#1700]
I am having the following error when I am running xspecfem3d command.

"Error nglob: sorted value 966017 differs from pre-computed 970023"

Can anyone help in this regards. Thanks in advance.

**Comment by aakash10gupta** [comment]
Hi @VarunD21029, it is likely a problem with the design of your mesh. Do you see the error on running xmeshfem3D or xspecfem3D? Please double-check, and report back with relevant input data files if you need further assistance, else close this issue.

**Comment by VarunD21029** [comment]
> Hi @VarunD21029, it is likely a problem with the design of your mesh. Do you see the error on running xmeshfem3D or xspecfem3D? Please double-check, and report back with relevant input data files if you need further assistance, else close this issue.

Yes you are correct it is error in xmeshfem3D command. I am unable to figure out the error. I am attaching a screenshot if that helps. Thanks for your reply
![specfem_error](https://github.com/SPECFEM/specfem3d/assets/95912657/2d72324f-fb49-4536-88a1-3acf2581972f)

**Comment by aakash10gupta** [comment]
Hi @VarunD21029, it is likely a problem with the design of your mesh. Please double-check your mesh design inputs.

You can also try looking at the problematic mesh using ParaView or another relevant software. The problematic mesh file should be in the your databases folder defined by the `LOCAL_PATH` variable in your `Mesh_Par_file`, and will have a name similar to `mesh_debug.vtk`.

This code block might also be helpful - https://github.com/SPECFEM/specfem3d/blob/master/src/meshfem3D/create_meshfem_mesh.f90#L763-L798

**Comment by VarunD21029** [comment]
Issue is resolved. I defined interface 1 at shallower depth than interface 2. Thanks for your help and sorry for the trouble

**Comment by aakash10gupta** [comment]
@VarunD21029, good to hear that you were able to resolve the issue. If you can, please change `xspecfem 3d` to `xmeshfem3D` in the title, and then close this issue. Thanks!
